templates/components/llamaindex/typescript/streaming/events.ts

```diff
@@ -6,23 +6,53 @@ import {
   ToolCall,
   ToolOutput,
 } from "llamaindex";
+import { LLamaCloudFileService } from "./service";
 
-export function appendSourceData(
+async function getNodeUrl(metadata: Metadata) {
```

Comment:
@thucpn this function is now at the bottom of the file (as it's not exported) - merge it there please